jsk_perception/sample/sample_people_pose_estimation_2d.launch

```diff
@@ -0,0 +1,108 @@
+<launch>
+
+  <arg name="gui" default="true" />
+  <arg name="GPU" default="-1" />
+  <arg name="INPUT_IMAGE" default="/camera/rgb/image_rect_color" />
+  <arg name="INPUT_DEPTH_IMAGE" default="/camera/depth_registered/hw_registered/image_rect_raw" />
+  <arg name="INPUT_CAMERA_INFO" default="/camera/rgb/camera_info" />
+  <arg name="INPUT_CLOUD" default="/camera/depth_registered/points" />
+
+  <include file="$(find jsk_perception)/sample/include/play_rosbag_people.xml" />
+
+  <arg name="MANAGER" value="people_pose_estimation_2d_nodelet_manager" />
+  <node name="$(arg MANAGER)"
+        pkg="nodelet" type="nodelet"
+        args="manager"/>
+
+  <node name="people_pose_estimation_2d"
+        pkg="jsk_perception" type="people_pose_estimation_2d.py"
+        output="screen">
+    <remap from="~input" to="$(arg INPUT_IMAGE)" />
+    <remap from="~input/info" to="$(arg INPUT_CAMERA_INFO)" />
+    <remap from="~input/depth" to="$(arg INPUT_DEPTH_IMAGE)" />
+    <rosparam subst_value="true">
+      gpu: $(arg GPU)
+      model_file: $(find jsk_perception)/trained_data/pose_estimation_2d_chainermodel.pkl
+      with_depth: true
+      scales: [0.38]
+      stride: 8
+    </rosparam>
+  </node>
+
+  <node name="people_mask_publisher"
+        pkg="jsk_perception" type="people_mask_publisher.py"
+        output="screen">
+    <remap from="~input" to="$(arg INPUT_IMAGE)" />
+    <remap from="~input/pose" to="people_pose_estimation_2d/pose_2d" />
+    <rosparam>
+      person_indices: 0
+      limb_part: RHand
+    </rosparam>
+  </node>
+
+  <node name="apply_mask_image"
+        pkg="nodelet" type="nodelet"
+        args="standalone jsk_perception/ApplyMaskImage" >
+    <remap from="~input" to="$(arg INPUT_IMAGE)" />
+    <remap from="~input/mask" to="people_mask_publisher/output" />
+    <rosparam>
+      clip: false
+    </rosparam>
+  </node>
+
+  <include file="$(find opencv_apps)/launch/hsv_color_filter.launch" >
```

Comment:
@iory I cannot see `hsv_color_filter.launch` in this directory. Will you create another pull request to `opencv_apps`?